src/lib.rs

```diff
@@ -90,26 +101,18 @@ impl Plugin for PointCloudPlugin {
             .init_resource::<SpecializedRenderPipelines<EyeDomePipeline>>();
 
         render_app
-            .add_systems((prepare_animated_assets,).in_set(RenderSet::Prepare))
+            .add_systems(
+                Render,
+                (prepare_animated_assets,).in_set(RenderSet::Prepare),
```

Comment:
These singleton tuples are annoying, can't remember if there is an alternative.

Reply:
I'm pretty sure they aren't required and it was just me doing it a bit too quickly and not updating it😅. I removed it and it still compiles